boards/arm/frdm_kw41z/Makefile.board

```diff
@@ -2,4 +2,6 @@ DEBUG_SCRIPT = jlink.sh
 
```

Comment:
It would be good to update the documentation to talk about different OpenSDA firmware needed for jlink.sh and pyocd.sh